.travis.yml

```diff
@@ -21,18 +21,15 @@ matrix:
       env: TOXENV=pypy3
     - env: TOXENV=flake8
     - env: TOXENV=checkspelling
```

Comment:
What do you think about running flake8 and checkspelling in a single build job?
(`TOXENV=flake8,checkspelling`)